action.yml

```diff
@@ -93,6 +93,7 @@ runs:
     - run: |
         : enable Cargo sparse registry
         echo CARGO_UNSTABLE_SPARSE_REGISTRY=true >> $GITHUB_ENV
+        echo REGISTRIES_CRATES_IO_PROTOCOL=sparse >> $GITHUB_ENV
```

Comment:
I think it should be `CARGO_REGISTRIES_CRATES_IO_PROTOCOL` according to https://blog.rust-lang.org/inside-rust/2023/01/30/cargo-sparse-protocol.html?

Reply:
I think you're right -- I got that from https://github.com/rust-lang/cargo/pull/11224 (you can see it was edited to correct it a day ago 😂 )